src/detector.ts

```diff
@@ -193,9 +203,13 @@ function logEvent(info: ProductEvent, node: Node) {
 
 function getId(event: ProductEvent): string {
   const items = JSON.stringify(event.items || []);
-  return [event.page, event.type, event.product ?? event.additionalProduct, event.bid, items].join(
-    "-",
-  );
+  return [
```

Comment:
It seems you may have different formatting settings. Could we revert all those changes.